lwatools/imaging/image_model_fitting.py

```diff
@@ -3,28 +3,78 @@
 import argparse
 import numpy as np
 import h5py
+import pickle
 from datetime import datetime
+import matplotlib.pyplot as plt
+
 from lsl.common import stations
 from lsl.reader.ldp import LWASVDataFile
 from lsl.imaging.data import VisibilityDataSet, PolarizationDataSet
 from lsl.imaging.analysis import find_point_sources
 from lsl.imaging.utils import build_gridded_image, plot_gridded_image
 from lsl.sim import vis as simVis
-from visibility_models import point_source_visibility_model_uv
-import pickle
-
 # from lsl.writer import fitsidi
 # from lsl.correlator import fx as fxc
-import matplotlib.pyplot as plt
 
-from imaging_utils import lm_to_ea, flatmirror_height, get_gimg_max
-from generate_visibilities import compute_visibilities_gen, select_antennas
-import known_transmitters
+from lwatools.file_tools.results import setup_results_h5
+from lwatools.vis_modeling.visibility_models import point_source_visibility_model_uv
+from lwatools.imaging.imaging_utils import lm_to_ea, flatmirror_height, get_gimg_max
+from lwatools.vis_modeling.generate_visibilities import compute_visibilities_gen, select_antennas
+from lwatools.utils import known_transmitters
+
+def grid_visibilities(bl, freqs, vis, target_bin, jd, valid_ants, station, size=80, res=0.5, wres=0.10):
+    '''
+    Resamples the baseline-sampled visibilities on to a regular grid. 
+
+    arguments:
+    bl = pairs of antenna objects representing baselines
+    freqs = frequency channels for which we have correlations
+    vis = visibility samples corresponding to the baselines
+    target_bin = the bin we want to use
+    jd = the date (shouldn't actually be important, but VisibilityDataSet needs it)
+    valid_ants = which antennas we actually want to use
+
+    returns:
+    gridded_image = 
```

Comment:
for all these arguments might be useful to add the dtype as well. I found it quite helpful having comments strewn through my scripts saying "This is a `lsl.imaging.data.VisibilityDataSet`" for example.